<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AsmForDos</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="mainGrid">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="mainTop">
      <item>
       <layout class="QVBoxLayout" name="topLeft">
        <item>
         <layout class="QHBoxLayout" name="fileOperations">
          <item>
           <widget class="QPushButton" name="openButton">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>27</height>
             </size>
            </property>
            <property name="text">
             <string>Open</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveButton">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>27</height>
             </size>
            </property>
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveAsButton">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>27</height>
             </size>
            </property>
            <property name="text">
             <string>Save As</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="closeButton">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>27</height>
             </size>
            </property>
            <property name="text">
             <string>Close</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTabWidget" name="codeTab">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="codeTabLayout">
           <attribute name="title">
            <string>Code</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="CodeEditor" name="codeTextEdit"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="topRight">
        <item>
         <layout class="QHBoxLayout" name="runOperations">
          <item>
           <widget class="QPushButton" name="runButton">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>27</height>
             </size>
            </property>
            <property name="text">
             <string>Run</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="compileButton">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>27</height>
             </size>
            </property>
            <property name="text">
             <string>Compile</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="compileAndRunButton">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>27</height>
             </size>
            </property>
            <property name="text">
             <string>Compile and Run</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTabWidget" name="resultTab">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="compilationTab">
           <attribute name="title">
            <string>Compilation Table</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QTableView" name="tableView"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="mainBottom">
      <item>
       <widget class="QTabWidget" name="outputTab">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="outputTabLayout">
         <attribute name="title">
          <string>Output</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QTextEdit" name="outputTextEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="errorTabLayout">
         <attribute name="title">
          <string>Errors</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QTextEdit" name="errorTextEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="options">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Options</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="clearButton">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>27</height>
            </size>
           </property>
           <property name="text">
            <string>Clear output</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>27</height>
            </size>
           </property>
           <property name="text">
            <string>Working directory:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLineEdit" name="workingDirectoryLineEdit">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>27</height>
              </size>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="workingDirectoryPushButton">
             <property name="minimumSize">
              <size>
               <width>25</width>
               <height>27</height>
              </size>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CodeEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>ui/codeeditor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
